app/client/src/components/pages/Community/components/tabs/Protect.js

```diff
@@ -181,40 +208,110 @@ function Protect() {
     protectedAreasHighlightLayer,
     waterbodyLayer,
     cipSummary,
-  } = React.useContext(LocationSearchContext);
-
-  const { infoToggleChecked } = React.useContext(CommunityTabsContext);
-
-  const sortedGrtsData =
-    grts.data.items && grts.data.items.length > 0
-      ? grts.data.items
-          .sort((objA, objB) => {
-            return objA['prj_title'].localeCompare(objB['prj_title']);
-          })
-          .filter(
-            (project) =>
-              project.ws_protect_ind && project.ws_protect_ind === 'Y',
-          )
-      : [];
+    allWaterbodiesLayer,
+  } = useContext(LocationSearchContext);
+
+  const { infoToggleChecked } = useContext(CommunityTabsContext);
+
+  const [normalizedGrtsProjects, setNormalizedGrtsProjects] = useState([]);
+
+  // normalize grts projects data with attains plans data
+  useEffect(() => {
+    if (grts.status === 'fetching' || grts.data.items.length === 0) return;
+
+    const grtsProjects = grts.data.items
+      .filter(
+        (project) => project.ws_protect_ind && project.ws_protect_ind === 'Y',
+      )
+      .map((project) => ({
+        source: 'grts',
+        title: project.prj_title,
+        id: project.prj_seq,
+        pollutants: project.pollutants,
+        total_319_funds: project.total_319_funds,
+        project_start_date: project.project_start_date,
+        status: project.status,
+        project_link: project.project_link,
+        watershed_plans: project.watershed_plans,
+        completionDate: '',
+        documents: [],
+        organizationIdentifier: '',
+        organizationName: '',
+        organizationTypeText: '',
```

Comment:
@cschwinderg  we should be consistent in using camelCase vs underscores in the property names – I'd suggest camelCase, as that's what we've used for all variable names across the project.

```suggestion
        source: 'grts',
        title: project.prj_title,
        id: project.prj_seq,
        pollutants: project.pollutants,
        total319Tunds: project.total_319_funds,
        projectStartDate: project.project_start_date,
        status: project.status,
        projectLink: project.project_link,
        watershedPlans: project.watershed_plans,
        completionDate: '',
        documents: [],
        organizationIdentifier: '',
        organizationName: '',
        organizationTypeText: '',
```

Reply:
@cschwinderg no action on your part right now – I'll push up an update for this.

Reply:
Makes sense. I committed the same change around the same time as you. It doesn't look like that merge broke anything.

Reply:
Haha, I saw that – no worries... that's partially my fault – I meant to start a review and accidentally clicked leave a single comment. It's just double fixed now :)